resilience4j/service2/src/main/resources/application.properties

```diff
@@ -15,10 +15,14 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-# the name of Camel
-camel.springboot.name=Service2
-camel.springboot.main-run-controller=true
-
-# the port number the service will use for accepting incoming HTTP requests
-service2.port=7070
-
+- routeConfiguration:
```

Comment:
yaml in a properties file?

Reply:
Fixed, reverted to previous application.properties file